src/components/Email/Email.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Email } from './Email';
+
+export default {
+  component: Email,
+} as ComponentMeta<typeof Email>;
+
+//“template” of how args map to rendering
+const Template: ComponentStory<typeof Email> = (args) => <Email {...args} />;
+
+export const MyEmail = Template.bind({});
```

Comment:
Let's name the main story as `Default`? Then when we have different variations, we can define them with the variant name? 

For example. For buttons, 
1. Default
2. Secondary (  Secondary button variation )

Reply:
yeah, sounds cool